### DISCUSSION TITLE
Global config vs User/Repo config

### DISCUSSION THREAD

**Discussion by BenWhiting** [renovatebot/renovate#17824] [Request Help] [answered]
After looking at the documentation I was not able to find this information, so sorry if I just missed it.

If I was self hosting Renovate and provide it a global config and it runs against a repo with a user/repo config present, what is the behavior?

I think it would attempt to merge the two configuration.

If the behavior is Renovate would merge the two configs, is there anyway to force the global config to always have priority over the user/local config when there is conflicting configuration?

**Comment by BenWhiting** [comment, marked as answer]
I just saw the option force which is global config only. 

https://github.com/renovatebot/renovate/blob/ea5806887b81b9800b941656b139e0333062f8fb/lib/config/options/index.ts#L243

Is this the expected way of achieving my usecase ?

**Reply by rarkins** [reply]
You can completely ignore user config using requireConfig=ignored. But otherwise what setting do you want to force?

**Reply by BenWhiting** [reply]
I saw I could do a complete ignore with that settings. I was hoping for a hybrid approach when it comes to create a configuration set. Some settings coming from global and others from user/repo, but still being able to override settings from the user/repo from the global config. 

A concreate example of this would be:
If the global config has the setting prFooter: "Please view this page for updates" and the user/repo has setting prFooter:"" just to clear the footer, I would like to make sure that the global config has priority, but still let the user/repo configuration setup how they would like other settings.

**Reply by rarkins** [reply]
The force object would be ok for the example you give. It's not as suitable if you need to combine it with complex package rules

**Reply by BenWhiting** [reply]
Great thank you!
